customize.dist/src/less2/include/creation.less

```diff
@@ -165,6 +165,7 @@
                 //margin: 10px 0;
                 min-height: 28px;
                 line-height: 28px;
+                flex-flow: column;
```

Comment:
This line breaks the style when the users have templates stored in their drives:
![image](https://github.com/user-attachments/assets/05f80bb7-ecb5-44c4-ac4f-3ae268b2dace)
![image](https://github.com/user-attachments/assets/f09b0c28-8a61-4fab-ac20-c864feae1cf3)

It adds scrollbars and the arrow button to show the next template page is under the list instead of next to it